config/initializers/barong.rb

```diff
@@ -43,6 +43,7 @@
   config.set(:barong_geoip_lang, 'en', values: %w[en de es fr ja ru])
   config.set(:barong_session_name, '_barong_session')
   config.set(:redis_url, 'redis://localhost:6379/1')
+  config.set(:barong_captcha, 'disabled', values: %w[disabled recaptcha geetest])
```

Comment:
Validate the possible values or through warning and set to disabled

Reply:
We validate it already